src/utils/view-on-OS.ts

```diff
@@ -0,0 +1,13 @@
+//import {USER_BIO_SELECTOR} from "../config.json" assert { type: 'json' };
```

Comment:
refactor (blocking): remove dead code

Reply:
I intend to move all the literals like the URLs, selectors to a configuration file. This was a part of doing that. Will be done by today.